frontend/src/stories/components/Commons/CommonsPlay.stories.mdx

```diff
@@ -0,0 +1,20 @@
+import { Meta, Story, Canvas } from "@storybook/addon-docs/blocks";
+import { action } from "@storybook/addon-actions";
+import CommonsPlay from "main/components/Commons/CommonsPlay";
+import {currentUserFixtures} from "fixtures/currentUserFixtures"; 
+import commonsFixtures from "fixtures/commonsFixtures"; 
+
+<Meta
```

Comment:
This is written in mdx instead of js. I am not sure if we are trying to be fully consistent across the whole project or not but the other stories were written in js.